roles/get-sitename/tasks/main.yml

```diff
@@ -142,10 +170,12 @@
     site_id: "{{ possible_site_ids|first }}"
   when: possible_site_ids|length > 0
 
-# - name: Debug site_id
-#   debug:
-#     var: site_id
-#
-# - name: Acsf site name
-#   debug:
-#     var: acsf_site_name
+- name: Debug site_id
+  debug:
+  var: site_id
```

Comment:
This needs to be indented.

Reply:
I'm not sure how this made it in here like this. I blame Russian hackers.

Reply:
It is a good thing I caught it. This is how it all starts. Next thing you know, Kanye is president.